### DISCUSSION TITLE
How to fix black screen, if it happened during update of Rlink2 ?

### DISCUSSION THREAD

**Discussion by ohayyam** [cedricp/ddt4all#1229] [Q&A]
Thanks in advance for your comments and helps.

I have Talisman 2016 with Rlink2 version 7.0.24.136 updated by offical renault service with micom version 5534. After installation it worked perfect without any problem, few days later i downgrade it to 2.2.15.703 with usb then update to v3.x to test and compare V3 & V7.  When i downgrade to V3 i got some problems like air condition control screen did not pop up , ambient ligths blinking and no voice from radio speakers. I tried to solve problems with ddt4 but could not succes then i update it back to V7.0.24.136 without any problems.

Renault started official update for 2.x Radio Rlink2 vehicles to V7.0.24.166 last week in Turkey. I got related LGU files and update my radio from 7.0.24.136 to 7.0.24.166 without any problems. But my updates v7.166 micom version was 5534 but micom was 5616 for official update by renault service. Although there are no any faults or problems on my radio, i decided to update my micom version from 5534 to 5616. For this update i have to downgrade again to 2.2.19.300, then update my micom to 5616 with rlink 2.2.19.300. 

I have to make one more last step to update again from 2.2.19.300 to 7.0.24.166. with latest micom version 5616. I repated this step before succesfully but that time my micom version was 5534. During the 7.0.24.166 update with usb 3 or 4 times radio reset and restart and continue to update with yellow lines :)  Btt this time after first reset it did not restarted and BLACK SCREEN... i waited 15 minutes nothing happened then i decide to remove usb unplug it, plug back... reset radio... went to service for reconfiguration, etc...  Finally i try with ddt4 and i enter all datas of radnav 2.x  ECU touch parameters and display properties and all others that i have backup my values. İt was usefull before and my black screen was normalised but that time voice was working and thinny white strip on my 8.7" screen.  

BUT NOW NO VOICE NO THIN WHITE STRIP ON SCREEN NOTHİNG !!! Like my radio dead!  Air condition working, all hard buttons working and ligths on , my multisense joystick and buttons not working and no  ligths on them. 

I tried pyren also to fix all reset commands but not worked to fix problem. 

Any idea to fix my radio ?

**Comment by shrlnm** [comment]
Could you try to make bugreport?
Make a folder "bugreport" on a usb flash and insert it into r-link2. Take a while. If the android core was loaded then the file bugreport.txt should appear in that folder.

**Comment by ohayyam** [comment]
Usb /sd port not live. Telematique unit have error on clip and not possible to erase. No software on radio :)

**Comment by shrlnm** [comment]
Then android core is not loading and we need to find the way to boot it in recovery mode. I've never heard how to do it. 

Do you have micom 5616 in binaries or it was installed with online connected can clip?

**Comment by ohayyam** [comment]
Yes it is loaded with online clip and i could not make a copy during micom upgrading. I think it is directly write to ecu but can be on a laptop somewhere at temp directory. What is the type of micom upgrade file ( bin, zip, xml ??  i can search it on my laptop and i can mail you that suspicious files :)

**Comment by shrlnm** [comment]
It may be a file with name like those in REPROG/DATA1 folder, but I not sure.

**Comment by shrlnm** [comment]
I have some guess, that recovery may start if you reboot rlink2 3-4 times in a minute. You may try to do it with button "hard reset" in dd4all screens with interval 10-15 seconds. But it is only my guess. Not for sure

**Comment by ohayyam** [comment]
i tried to hard reset with 10 sec intervals 5 times nothing happened . then ?

**Comment by shrlnm** [comment]
I seen such clue in oldest firmware. Probably it was not relevant information.

**Comment by ohayyam** [comment]
i am mgoing to change tools now. i try to reprog radio ecu gives error 105. means ecu ref number not recognised. if we can write part number to raido with ddt then i can reprog it. Can we make this page editable ?
![ecu 105 cozumu ddt](https://user-images.githubusercontent.com/35384108/53082580-35cdba00-350e-11e9-9e86-5b9d3e750db2.png)

**Comment by ohayyam** [comment]
which file you are intresting of my laptop to check ?
![c scan ecu ddt log](https://user-images.githubusercontent.com/35384108/53087680-c610fc80-3518-11e9-923c-31a5237bc18e.png)

**Comment by shrlnm** [comment]
I do not know where the CLIP may store temporary files. Probably it may be found by time of creation

**Comment by ohayyam** [comment]
if it store on laptop should be on my c drive file list which i selected last hour of upgrading time both ddt and clip

**Comment by ohayyam** [comment]
i am going to close solution but need help as always :)))

here is the ss of clip ddt and pyren . shows that if we can write red box of ddt4 then clip can reprogram the my radio. is not it ?

![clip pyren ddt4 red box](https://user-images.githubusercontent.com/35384108/53262782-99abda80-36e8-11e9-90eb-254b88e3c42b.jpg)
![pyren ddt4 red box](https://user-images.githubusercontent.com/35384108/53262783-99abda80-36e8-11e9-9737-305ddefb70cc.jpg)
![ddt4 red box](https://user-images.githubusercontent.com/35384108/53262784-99abda80-36e8-11e9-95c6-3ed1466f0db0.jpg)

Thanks & Regards

**Comment by ohayyam** [comment]
> Could you try to make bugreport?
> Make a folder "bugreport" on a usb flash and insert it into r-link2. Take a while. If the android core was loaded then the file bugreport.txt should appear in that folder.

@shrlnm did i forget some folders ?


![tal full usb folders](https://user-images.githubusercontent.com/35384108/53265561-9c5dfe00-36ef-11e9-9d09-0b0ed90764ea.png)

**Comment by ohayyam** [comment]
@shrlnm is there any list for pyren responce codes meaning ?

![pyren resp codes](https://user-images.githubusercontent.com/35384108/53266210-2a86b400-36f1-11e9-80ea-d75146acc4c3.jpg)

**Comment by shrlnm** [comment]
> here is the ss of clip ddt and pyren . shows that if we can write red box of ddt4 then clip can reprogram the my radio. is not it ?

Not sure that we can change these values. 
Why Clip shows nothing?

**Comment by shrlnm** [comment]
> did i forget some folders ?

just folder "bugreport" should be enough

**Comment by shrlnm** [comment]
> is there any list for pyren responce codes meaning ?

it is not pyren response codes. It is ECUs response codes and the ddt4all shows the same.

On your example. You sent 3 commands. The first one  2E220101 consist of three part
1) 2E - CAN command WriteDataByLocalIdentifier
2) 2201 - LID (local identifier) the reference on a parameter
3) 01 - the value you are going to write to parameter with LID=2201

Now look on response - RSP:6E2201
6E  = 2E+0x40  - it is a positive response on command 2E. Then the LID=2201 follows. Actually it means that the command WriteDataByLocalIdentifier to LID 2201 was accepted and executed.

For two other commands we also received positive responses

1101 - ECU hard reset
1003 - open new diagnostic session

In case when something goes wrong during command execution you will receive NegativeResponse or NR. It always starts withs 7F byte followed by command's first byte and in third byte you will receive NR:code. The descriptions of all NR codes you may find in file ddt4all/elm.py

**Comment by ohayyam** [comment]
> > here is the ss of clip ddt and pyren . shows that if we can write red box of ddt4 then clip can reprogram the my radio. is not it ?
> 
> Not sure that we can change these values.
> Why Clip shows nothing?

clip looking all values. if some of them missing or can not read ( like red box of ddt box values )  dont continue to read and shows nothing :) Actually i was surprised how pyren can read values ?? DDT and clip could not read? 

Actually during update i unplug usb accidentaly :) normally nothing in ecu like formatted or infected and damaged by virusses of windows 10  :))

And my other 4 ecu have red errors and can nor erased by can clip and pyren. I really dont now. Errors are related with radio ecu damage? Because I checked all fusess and all ok.

**Comment by ohayyam** [comment]
> > did i forget some folders ?
> 
> just folder "bugreport" should be enough

i am collected all special folders on my USB. If we can fixed radio then i may need all of them :D

**Comment by ohayyam** [comment]
ECU trouble codes
DTC #0
DTCStatus.pendingDTC : 0 [0x0]
DTCDeviceIdentifier : [930b] Rear Right Loudspeaker
DTCStatus.testFailedSinceLastClear : [00] No
DTCDeviceAndFailureTypeOBD : 37643 [0x930b]
DTCFailureType.ManufacturerOrSupplier : 1 [0x1]
DTCFailureType.Category : [01] General Electrical Failures
DTCFailureType : [13] circuit open
DTCStatus.testNotCompletedSinceLastClear : [01] Yes
DTCStatus.testNotCompletedThisMonitoringCycle : 0 [0x0]
DTCStatusAvailabilityMask : 57 [0x39]
DTCStatus.warningIndicatorRequested : 0 [0x0]
StatusOfDTC : 16 [0x10]
DTCStatus.testFailedThisMonitoringCycle : 0 [0x0]
DTCStatus.testFailed : [00] No
DTCStatus.confirmedDTC : [00] No

DTC #1
DTCStatus.pendingDTC : 0 [0x0]
DTCDeviceIdentifier : [930b] Rear Right Loudspeaker
DTCStatus.testFailedSinceLastClear : [00] No
DTCDeviceAndFailureTypeOBD : 37643 [0x930b]
DTCFailureType.ManufacturerOrSupplier : 1 [0x1]
DTCFailureType.Category : [01] General Electrical Failures
DTCFailureType : [1c] circuit voltage out of range
DTCStatus.testNotCompletedSinceLastClear : [01] Yes
DTCStatus.testNotCompletedThisMonitoringCycle : 0 [0x0]
DTCStatusAvailabilityMask : 16 [0x10]
DTCStatus.warningIndicatorRequested : 0 [0x0]
StatusOfDTC : 16 [0x10]
DTCStatus.testFailedThisMonitoringCycle : 0 [0x0]
DTCStatus.testFailed : [00] No
DTCStatus.confirmedDTC : [00] No

**Comment by ohayyam** [comment]
my obdlink SX which has latest 4.0.1 fw installed but elmbenchmark 83/85 ok

what is it mean that 2 of them fail ???


v1.4 ATCEA [FAIL]
v1.4 ATCEA 01 [FAIL]

**Comment by gobo-ws** [comment]
OBDLink SX (based on STN11xx chip) does not support the AT command ATCEA according the following document:
https://www.scantool.net/scantool/downloads/234/stn1100-frpm-preliminary.pdf

ATCEA "Turn off CAN extended addressing" Status: not yet supported
ATCEA hh "Use CAN extended address hh" Status: not yet supported

**Comment by ohayyam** [comment]
@shrlnm would you like to connect mylaptop who is connected to my talisman with oblink sx with teamviewer. 
Pyren and ddt4 are also ready :dancer: 

![shrlnm ddtscreen radio](https://user-images.githubusercontent.com/35384108/53693296-a34fd500-3daf-11e9-9658-bd338f0f7abd.jpg)

**Comment by Sig2018** [comment]
@ohayyam Did you fix your problem?
I'm curios about how you upgraded MICOM?
Is ther any you can do with ddt or pyren?
Is there any limitation on Hardware for MIcon Upgrade? I've heard about 9 or 10 bits depend on version.

**Comment by ohayyam** [comment]
@Sig2018 My problem not fixed yet. My screen still blank and dead :)
Micom updated at service with clip under renault.net with token. (officially ugraded) During upgrade first part micom update completed then second stage R-Link2 update started but not finished gives an error. Then i stop process and reset my radio. Finally i got Rlink-2 version 2.2.19.300 and micom was 5615. I got V7.0.24.16x on usb and then try to update with only usb. It started to update and i did it before 2 times successfully but last time when it was reseting first time it did not open again. I wait 30 minutes nothing then remove USB and result my radio dead :)  Still looking for a solution like how to enter bootloader mode etc. I already mail to LG factory they not replied yet :( On the other hand i am looking to buy new radio with DAB+ANC+BOSE with nativeV7. Prices of radio at France 272 Euro, but noway to buy Still NO STOCK !!!

**Comment by Sig2018** [comment]
And you cannot ask service to help you load software again?
How did you convince them to upgrade Micom?

**Comment by ohayyam** [comment]
I was at service and i tried myself all combinations :) I damaged my radio before. They reject the replace under warranty second time :)

**Comment by Sig2018** [comment]
Ohh you used the service tool. It is is a pity you can find ghe problem.

**Comment by ohayyam** [comment]
#123 shows solution to test and check not tested yet :))

**Comment by kadjaf45** [comment]
Hello. please help at rlink. After upgrading to v7, I can't update my Renault Map store. After connecting ddt4all error.
Scan  CAN

Found ECU [BCM/UCH] (not perfect match) :CMF1_BCM_DDT_10 DIAGVERSION [08] SUPPLIER [CONTINENTAL AUTOMOTIVE ] SOFT [1426 ] VERSION [1030 instead 1000 ]
Identified ECU [UPC-EMM]@27 : USM_CMF1_Sailing_B4_20160126T144031.json DIAGVERSION [1]SUPPLIER [414] SOFT [DB43] VERSION [2700] {1963}
Found ECU [HVAC] (not perfect match) :ClimBoxGen2_v26 DIAGVERSION [4] SUPPLIER [740] SOFT [E010] VERSION [1622 instead 1619]
Identified ECU [Airbag-SRS]@2C : RC7_ACU_eng_v2.22_20180123T134644.json DIAGVERSION [08]SUPPLIER [255414 ] SOFT [R7_021XH.P82 ] VERSION [P82 ] {7467}
Found ECU [Cluster/TDB] (no relevant ECU file found) :DIAGVERSION [04] SUPPLIER [Visteon_Namestovo_096 ] SOFT [V0711V07] VERSION [V0711V07]
Identified ECU [Power steering]@04 : CMF1_CEPS_V1_10_20160225T124409.json DIAGVERSION [17]SUPPLIER [231] SOFT [0007] VERSION [0001] {7767}
Identified ECU [EPS]@E9 : CMF1_CEPS_V1_10_20160225T124409.json DIAGVERSION [17]SUPPLIER [231] SOFT [0007] VERSION [0001] {7767}
Found ECU [FCAM] (no relevant ECU file found) :DIAGVERSION [64] SUPPLIER [4G2] SOFT [0011] VERSION [6DDD]
Found ECU [Audio] (not perfect match) :RadNav 2.X v2.5 DIAGVERSION [0D] SUPPLIER [CAP] SOFT [70] VERSION [24131 instead 23906]
Found ECU [ECM/Injection] (not perfect match) :EMS3155_RD6_750_5C0_7D0_930_V01 DIAGVERSION [44] SUPPLIER [CONTINENTAL_ENGINE_SYSTEM] SOFT [00D6 ] VERSION [7500 instead 9300]
Identified ECU [DCM Renault]@77 : TCU_GEN2_V3.2_20151109T114222.json DIAGVERSION [8]SUPPLIER [284] SOFT [0880] VERSION [0411] {2914}

**Comment by shrlnm** [comment]
@kadjaf45 check if VIN written to ECU

**Comment by kadjaf45** [comment]
Yes written

**Comment by ohayyam** [comment]
@kadjaf45 what is your story , what did you do and what happened ? What was your rlink2 version and micom version before upgrade to V7 ?

**Comment by kadjaf45** [comment]
I had version 2.2.19.300, after upgrading to v7, there was a black screen in rlink. service replaced me with a new radio with V7 software. I can't update the maps right now. after inserting the pendrive into usb, the R-link and Lost folders are created. in folders of no .wls or lyc files. myrenault can't see my car. 
after entering the VIN VIN website I show the upgrade to v2.
![Screenshot_20191215-142027_Chrome](https://user-images.githubusercontent.com/58991703/71056952-55983380-215b-11ea-9163-8e66ef96ff2f.jpg)

**Comment by shrlnm** [comment]
> in folders of no .wls

You need to write a VIN number into R-LINK2. The service had to do it after replacement but probably they forgot.

**Comment by kadjaf45** [comment]
WHERE TO PLEASE ENTER IT VIN. I AM NOT VERY GOOD AT DDT.
if it is safe, the radio will not exchange for a second time on Renault. if I turned on my android car by ddt4all myself, they will find out about it.

**Comment by ohayyam** [comment]
@kadjaf45 no matter how good you are in DDT. You have to go to Renault service to write your vin number to your new radio and then system activation has to be carried out with token and clip under renault.net. Otherwise renault server never recognise your radio on your car and you will never get any update apk or map from the system. Your radio replacement done by under Renault warranty rigth ? and new radio is  proper for your car ?  If your radio replaced with another type of Radio then service can not do autoconfig and service activation. If you are in this case then you have to do autoconfig with DDT ( @shrlnm he may need your delicious macro file :)) at this stage for autoconfig.... then  radio need service activation but not for autoconfig at renault service ! Do you have replaced radio label photos ?

**Comment by kadjaf45** [comment]
autoconfig how to do it. what ro is macro. sorry but i'm incomplete in this matter.

**Comment by ohayyam** [comment]
Autoconfig can be carried out by 3 ways which one of them not good for you that clip autoconfig at service. Second way : You can use macro with pyren program ( you can ask this matter to @shrlnm ) 3 way: You can enter all your radnav ecu datas manually with ddt4all. In this case you need to find similiar car screenshots form ddt4all. If your car Talisman , you can download and use this SS.  https://yadi.sk/d/uSzolzhTpqKaRw

**Comment by kadjaf45** [comment]
in my renault kadjar my country is brazil and the maps of south america, and I'm from Poland in Europe, can this be the problem?

**Comment by AshvanR** [comment]
HI, i am searching for some help.
I think i used a wrong file to update my rlink 2 in my espace.
Screen is now black and usb not working.
Rlink is trying to start which i can see on my dashboard, it blinks a second and then restarts.
Is there a way to get my rlink back to life?

**Comment by kadjaf45** [comment]
I change the radio on the website.

Pozdrawiam Sławomir Golonka

________________________________

**Comment by Flavius08** [comment]
Can someone help me with the next problem ? Pleaseeee... diag has been set invalid value gen1_1
I went into Developer mode and changed 2 settings (first picture) to both it wrote off and I set as in the first picture, sports and comfort and after I saved and restarted the radio I got an error (picture 2) and  I can't do anything anymore, please if anyone can help me.
![20200621_000656](https://user-images.githubusercontent.com/67219974/85219302-e2d67200-b3a2-11ea-8a3b-d210ec342ae7.jpg)
![20200620_182420](https://user-images.githubusercontent.com/67219974/85219307-e964e980-b3a2-11ea-8c92-712d2901fa7e.jpg)

**Comment by estyasser** [comment]
@ohayyam Hi friend,

I have almost the same issue, except simpler. I upgraded from V2 to V7 by USB only and the screen went dead, it's in the boot loop, I can tell that from the instrument panel (where the gages are) keep flashing every 10-15 seconds.
Also AC is working and hard buttons are too, but no sound no radio no screen.

**Have you managed to find a solution to that?**

The service center here are dump and said the system is locked and we can't access anything or know what happened therefore refused to fix the issue and said we can't reflash the firmware, they only offered me to change the radio unit for $3100 which so expensive.

**Comment by alireza6631** [comment]
Hello comrades, you were able to find a solution, because I also have your problem

**Comment by TheApostolos** [comment]
Hello to everyone. I am new here.

I bought before a week a Renault Kadjar 1.2 tce mod.2016. It had version: 2.2.15.703. It had problem connecting with my Samsung Fold3 so i decided (wrong decision, finally) to update it with this process.

I manage to finish the update. The version now is: 8.0.32.501. It is working...but:

- The thermometer fails. It is shows dashes and F instead of C like before.
- Parking sensors are dead (although in the settings are ON)
- The ECO app doesn't work. It is shown dashes
- Lane Assistant doesn't make sound
- My phone is connected and it works fine but i can hear voice in calls. (Music plays perfect).
- The contrast light on the dash monitor and on the main monitor is setting automatically to 30%. Although i set it to 100% it going back to 30% after some secs.


I didn't like all that and i made a second serious mistake and through the development mode i pushed the virgination button. The R-Link2 stop working (black screen at all)

I went to a dealer and he managed to set it on again but in 8.0.32.501 again with the same problems and 2 more. One is that the R-Link Store does not connect. And the second is that when I am inserting a blank USB in order to put it in my PC (Toolbox), it creates one file and an empty Folder called R-Link. If I press the ON/OFF button to reset it, the monitor stop working. (The garage tried that).

I 've made an appointment to the central Renault but they told that they don't know what is going on and they made a question to the Renault central factory if they have a patch. 

If you have any idea, is more than welcome.

So be careful with that.

**Comment by Flavius08** [comment]
My uncle repaired it with a device auto scan, he reset it. On 13.06.2022, 23:19 maiiikell1995 ***@***.***> wrote:

   
  @TheApostolos did you get it fixed by renault?

**Comment by CelviD** [comment]
> Hello to everyone. I am new here.
> 
> I bought before a week a Renault Kadjar 1.2 tce mod.2016. It had version: 2.2.15.703. It had problem connecting with my Samsung Fold3 so i decided (wrong decision, finally) to update it with this process.
> 
> I manage to finish the update. The version now is: 8.0.32.501. It is working...but:
> 
> * The thermometer fails. It is shows dashes and F instead of C like before.
> * Parking sensors are dead (although in the settings are ON)
> * The ECO app doesn't work. It is shown dashes
> * Lane Assistant doesn't make sound
> * My phone is connected and it works fine but i can hear voice in calls. (Music plays perfect).
> * The contrast light on the dash monitor and on the main monitor is setting automatically to 30%. Although i set it to 100% it going back to 30% after some secs.
> 
> I didn't like all that and i made a second serious mistake and through the development mode i pushed the virgination button. The R-Link2 stop working (black screen at all)
> 
> I went to a dealer and he managed to set it on again but in 8.0.32.501 again with the same problems and 2 more. One is that the R-Link Store does not connect. And the second is that when I am inserting a blank USB in order to put it in my PC (Toolbox), it creates one file and an empty Folder called R-Link. If I press the ON/OFF button to reset it, the monitor stop working. (The garage tried that).
> 
> I 've made an appointment to the central Renault but they told that they don't know what is going on and they made a question to the Renault central factory if they have a patch.
> 
> If you have any idea, is more than welcome.
> 
> So be careful with that.

Im having the exact same issue. My father used kadjar upgrade file in his Megane and now the RLink2 is exacatly like yours. How did you solve it? Thanks

**Comment by ubem23** [comment]
hello everyone,
I join the discussion I switched from 2 to 7 then by mistake I inserted the key with 8 happened an errero and now I am trying every way to go back to 2 but any key with any fw inside returns me yellow writings that I list here and then performs 3 resets then restarts the radio with impossible to update and no longer work parking sensors air conditioning etc... can any of you help me????

thank you very much

[mainupdate_rotation][848] ro.sf.hwrotation = 270
[uart_bootnotify][220]send IPC 1
[uart_bootnotify][223]send IPC 2
[uart_bootnotify][226]send CAN central display activation IPC
[miscpartition_setmode][285]theader.mode : [0]
[main_update][991]start main_update (Normal download)
[mainupdate_mountfatfs][492]blkpartnum[/dev/block/sdb1][1]
[miscpartition_getusbid][278]miscpartition_getusbid : f0344afa
[mainupdate_mountfatfs][503]usb id : [f0344afa]
[mainupdate_getusbFatId][206]fat_type=185[mainupdate_getusbFatId][242]Unknown filesystem signature: , path=/dev/block/sdb
[mainupdate_getusbFatId][206]fat_type=0[mainupdate_getusbFatId][230]id=f0344afa
[mainupdate_mountfatfs][523]origianl [/dev/block/sdb1] change usb block to [/dev/block/sdb1]
[mainupdate_mountfatfs][531]USB BLK NAME DETECTING
[mainupdate_mountfatfs][549]we can find USB(/dev/block/sdb1) util waitcount: 0
[mainupdate_mountfatfs][550]we will try mount USB(/dev/block/sdb1) /vold
[mainupdate_mountfatfs][576]we will try mount SDCARD(/dev/block/mmcblk0p4) /sdcard
[mainupdate_mountfatfs][593]we are done for mounting
[mainupdate_checkHAB][936] [ Secured H/W ]
[mainupdate_makefilepath][683]blkname (/dev/block/sdb1) pfilepath (/vold/R-LINK/mm2014_factory.lgu) filename (R-LINK/mm2014_factory.lgu)
[mainupdate_avoidchrckcrc][835]/vold/R-LINK/mm2014_factory.lgu will skip its CRC checking when it is opened
[parser_openupgradefile][846]len = 2, 11
[parser_openupgradefile][848]Finger print: updateall _gen11/mm2014_gen11:4.0.4/UNKNOWN/.201212.255454:user/test-keys svn=3256|252665
[feature_getfingerprint][44]feature_getfingerprint (44)
[feature_getfingerprint][162]feature_getfingerprint (162): Start force update
[parser_matchfingerprint][284]Version info: devfingerprint [updateall]
[parser_matchfingerprint][285]Version info: headfingerprint [updateall]
[parser_matchfingerprint][295]Update all
[miscdualboot_initupgrade][132]MISC DUAL update index 1, usermode : 0
[miscpartition_setupgradestatus][202]result flashcopy
[mainupdate_calculatetotaltime][823]mainupdate_calculatetotaltime timesec : 767433
[miscpartition_setupgradestatus][202]result usbcopy
[miscpartition_incbootcount][298]theader.recoverybootcount : [4]
[main_update][1132]Recovery Boot Count : [4], haveimagetype=0, getimagetype=0
[main_update][1136]upgradeimgs : 16 curimg : 0
[mainupdate_upgradelogprepare][386] Upgrade log files index : 0
[main_update][1169]file_name[/vold/R-LINK/mm2014_factory.lgu] partitionidx[2]
[upgradeimage][684]partition index[0x02] offset:0x19b
[upgradelargeimage][208]partition index[0x02] offset:0x19b
[upgradelargeimage][236]not large file
[progress_log][44][1/16] PROGRESS - CURIMAGE:  0|TOTAL:  0, REMAINING TIME(min):13, totalms : 767433
[mirror_partition][199]mirror_partition 2
[mirror_partition][213]Read from /dev/block/mmcblk0p2
[mirror_partition][248]Write to /dev/block/mmcblk0p11
[mirror_partition][275]Read from /dev/block/mmcblk0p11
[upgradeimage][998]Success recovery mirror to 0
[upgradeimage][1002]Error: Recovery partition change to 11: -159
[main_update][1188]main update error partition [2]
[mainupdate_copyupgradelogfile][427]copy upgrade log file 0 !! 
[mainupdate_copyupgradelogfile][431] Create upgrade log directory : /vold/upgrade.log

**Comment by AlMeu** [comment]
@CelviD yours unit is bricked, now you have to replace the unit with one new.

@ubem23 yours unit have memory problem failing to write and also you need to replace with another new.

I can help with new units if you like. Who wants leave an email for contact.

**Comment by ubem23** [comment]
nooooooooo but if I get it used I was reading about anti-theft problems can it be fixed?

**Comment by AlMeu** [comment]
@ubem23 yes, can be unlocked, but only if I get it on table.
An email and we can talk.

**Comment by Sergtopic** [comment]
> @ubem23так, можна розблокувати, але лише якщо я отримаю його на столі. Електронна пошта, і ми можемо поговорити.
Hi AlMeu. If I provide a dump of EEPROM 24c256 and VIN and ID, will you be able to adjust the dump for my car?

**Comment by AlMeu** [comment]
@Sergtopic I will try and see. Leave an email and we will speak.

**Comment by Sergtopic** [comment]
My email address is slusarsa@gmail.com

**Comment by maui3323** [comment]
Hello, I have very similar problem. Tried to update to rlink 3.0.x on a 2017 renault megane and now the screen is black as well as no usb and no radio. Can someone help me please? maui3323@gmail.com

**Comment by dimdarzov** [comment]
> @SergtopicЯ постараюсь и посмотрю. Оставьте письмо, и мы поговорим.

Good afternoon. is it possible to link the radio from another car? dimdarzov@gmail.com

**Comment by Sertan-cotech** [comment]
hello, i have the same problem.  Can another vehicle's radio be connected?
sertanerman.se@gmail.com

**Comment by matzel82** [comment]
Normally SW with Original Diagnostic Tool. I know this problem. I'm working at a Renault Garage. It happens to me sometimes. Update, Restart, Config and powerlatch... normally no problem for a real Renault Expert

**Comment by Sertan-cotech** [comment]
First of all, thank you for your interest.  I also work in a Renault garage
in Türkiye.  I am encountering these problems.  now I need to match a
product that was removed from another vehicle to a customer's vehicle.  The
problem is that the device introduced earlier with the clip diagnostic
interface cannot be adapted to another vehicle.  Vin and ID do not match.

6 Haz 2023 Sal 21:21 tarihinde matzel82 ***@***.***> şunu
yazdı:

**Comment by matzel82** [comment]
That wouldn't be work. You have to order a new Navi-Unit behind the display. Order with VIN

**Comment by shrlnm** [comment]
> hello, i have the same problem. Can another vehicle's radio be connected?

If anti-theft blocking was enabled on the box from donors car then it will blocked when you connect RL2 from other car to the CAN bus of your car.

There is some gossip in the network that anti theft blocking may be removed temporary by manipulation with pins on a big pinout of RL2. Consequent virginizing of the box remove blocking permanently, but I do not know details.
We may also temporarily remove anti-theft blocking by simply arduino based can gateway which will replace vehicleid in one of system frame, it allows us to run a box, get in to developer mode and virginize a box for permanently remove blocking.

**Comment by Sergtopic** [comment]
There is a known method of unlocking the radio by replacing the ID and VIN
of your car in the flash memory.

ср, 7 черв. 2023 р. о 16:21 shrlnm ***@***.***> пише:

**Comment by shrlnm** [comment]
@Sergtopic yes I agree. It's third method

**Comment by rafal1912** [comment]
Hi,

I am new here. Car grand scenic 2016, I updated rlink2 ver 7 to rlink2  ver 8 with no problem. Next I started to do the same from ver 8 to ver 9, with the same stick. Evrything looked the same as in previous update but the screen did not start, radio did no start, hard buttons work, AC works.

Please advise.
thanks!

**Comment by matzel82** [comment]
The problem is...that some RL not make for newer SW....

**Comment by rafal1912** [comment]
> The problem is...that some RL not make for newer SW....

SW?

**Comment by matzel82** [comment]
SW=Software...

**Comment by AlMeu** [comment]
@rafal1912 
- there is sound in speakers?
- USB are working?
- the dashboard is flashing in 2 colors?

**Comment by rafal1912** [comment]
> @rafal1912
> 
> * there is sound in speakers? **No**
> * USB are working? **Phone is charging** **but car does not wrine any file on new stick**
> * the dashboard is flashing in 2 colors? **Yes**

**Comment by AlMeu** [comment]
@rafal1912 
your unit is dead and remains only to change it with other, but be carefull.
As you see in other discussions all units installed in these cars are locked to work only in that car.
Maybe I can help you if you look for other unit.

**Comment by rafal1912** [comment]
> @rafal1912 your unit is dead and remains only to change it with other, but be carefull. As you see in other discussions all units installed in these cars are locked to work only in that car. Maybe I can help you if you look for other unit.

by unit you mean new radio modul? like this from below
https://allegro.pl/oferta/oryg-nowe-radio-nawigacja-renault-kadjar-megane-iv-13383543413

??

**Comment by AlMeu** [comment]
Yes, the unit like that.

**Comment by pm8009** [comment]
Hi,

I am new here. Car Renault Talisman 2016, I updated rlink2 ver 2 to rlink2 ver 3 with the same stick. Evrything looked normal but the screen did not start, radio work, hard buttons work, AC works. On the screen I get this message: 
Diag has been set invalid value. Try again
Configuration
HMI.AmbilightSetting has invalid value : 0
MEX.CF_Ambient_Lighting_Presence has invalid value : 1
HMI.MassingSeat has invalid value : 0
MEX.CF_MassingSeat_Presence 
![Uploading IMG_20230608_191936.jpg…]()
has invalid value : 1

Please advise.
thanks!

**Comment by matzel82** [comment]
You're a problem dad every car have to same hardware, but a different software, not every car have Ambilight or Message seats

**Comment by pm8009** [comment]
Thank you for information.
Please tell me how I can remove these inscriptions from the screen and start the system? I still have the previous working firmware on Usb flash. Maybe it is possible to start an emergency download of the old firmware? The version of my bootloader is 4992. And I have a small screen.
(Pressed for few minutes buttons "volume up" + "volume down" + "touch home" + "touch screen OFF" did not help)

**Comment by AlMeu** [comment]
@pm8009 
    - there is sound in speakers?
    - the USB are working?
    - the dashboard is flashing in 2 colors?

**Comment by pm8009** [comment]
- there is sound in speakers?
Yes.
- the USB are working?
Yes.
- the dashboard is flashing in 2 colors?
Yes (white inscriptions on black screen).
I am still have the previous working firmware on Usb flash. Maybe it is possible to start an emergency download of the old firmware? The version of my bootloader is 4992. And I have a small screen.
(Pressed for few minutes buttons "volume up" + "volume down" + "touch home" + "touch screen OFF" did not help)

**Comment by AlMeu** [comment]
Then you are lucky.
Fix these settings which is telling that are invalid and perhaps you get unit back in running:
"_HMI.AmbilightSetting has invalid value : 0
MEX.CF_Ambient_Lighting_Presence has invalid value : 1
HMI.MassingSeat has invalid value : 0
MEX.CF_MassingSeat_Presence has invalid value : 1_"

**Comment by pm8009** [comment]
Please tell me in which section of the system these parameters are located and which program is better to change them: Ddt4all or Pyren?
And is it right to change these parameters to reverse values?:
HMI.AmbilightSetting has invalid value : 1
MEX.CF_Ambient_Lighting_Presence has invalid value : 0
HMI.MassingSeat has invalid value : 1
MEX.CF_MassingSeat_Presence has invalid value : 0

**Comment by rafal1912** [comment]
> Yes, the unit like that.

Thanks, do you know where in grand scenic 2016 this unit is located?

**Comment by AlMeu** [comment]
@rafal1912 I think in Scenic IV the unit is above glove box like Espace V.

@pm8009 with interface you have never will you be able set individual those parameters.
Use ddt4all

**Comment by Cocobep** [comment]
> Thực hiện một thư mục "bugreport"

Good day.

Please show me how to create a bug report file

Thanks for the help

**Comment by kuru0777** [comment]
Merhaba ben de clio4 aracın media nav evo2 multimedya cihazımı sürüm yükseltirken çökerttim güncelleme bitti ve siyah ekran hiç bi şey açılmıyor sorunun çözümünü biliyor musunuz?

**Comment by Vengo222** [comment]
Hi... How can I update from version 2 to version 7 on a Renault Laguna with R-link 2?

I have Can-Clip and the LGU update file; I don't have the MICON update file, or any other files that might be needed.

Any help would be appreciated.
